gpbft/participant.go

```diff
@@ -161,8 +161,9 @@ func (p *Participant) ReceiveMessage(vmsg ValidatedMessage) (err error) {
 
 	// Drop messages for past instances.
 	if msg.Vote.Instance < p.currentInstance {
-		return fmt.Errorf("message %d, current instance %d: %w",
-			msg.Vote.Instance, p.currentInstance, ErrValidationTooOld)
+		p.tracer.Log("dropping message from old instance %d while received in instance %d",
```

Comment:
Do we want to signal this to upstream instead by non-critical error typing?

Reply:
> Do we want to signal this to upstream instead by non-critical error typing?

We could add proper level-based logging, is that what you're suggesting?

Reply:
That too. What I was suggesting is introducing a `RetriableError` type which the caller can check if a returned error is a type of. So we continue to return error but type it based on whether it is retriable, i.e. non-critical, or not. The integrator then chooses to do whatever they wish with it.

Reply:
This is a non-blocker btw; we can add this later if upstream actually finds it useful.